src/sentry/static/sentry/app/views/organizationDashboard/dashboard.jsx

```diff
@@ -34,5 +34,7 @@ const Widgets = styled('div')`
 `;
 const WidgetWrapper = styled('div')`
   width: 50%;
-  padding: ${space(1)};
+  :nth-child(odd) {
+    padding-right: ${space(2)};
```

Comment:
shouldn't this be `margin-right`?

Reply:
It's padding because the wrapper's width is 50% (and because the container is a flexbox with flex wrap)